prdiff.go

```diff
@@ -58,7 +58,7 @@ func (t *prdiff) GetDiffPullRequests(ctx context.Context, base, head string) ([]
 	prs, err := t.getPullRequestsMergedBetween(
 		ctx,
 		cc.GetBaseCommit().GetCommit().GetCommitter().GetDate(),
-		time.Now(),
+		cc.Commits[cc.GetTotalCommits()-1].GetCommit().GetCommitter().GetDate(),
```

Comment:
I'll give you that. My mistake. 🙇 🙏 
But, this code will not work well.[*1]
Because most timestamp of merge commit is **before** merging of pull request. (rarely, it is equal)
I think we have to update additional code below.
1. +1 seconds to last commit date. (GitHub returns second precision timestamp)
2. make the condition to closed-interval (`[t1, t2]`)
https://github.com/storz/prdiff/pull/2/files#diff-9ba84b6b4500a897458d3976afde5bb2R100

I think this is NOT completely accurate, but better than now.
[1] Please try specifying merge commit as head.

Reply:
@storz I'm sorry for late reply. I fixed it and rebased. Please take a look :bow:

Reply:
@susho Thank you for fixing.
> make the condition to closed-interval ([t1, t2])
How do you think about this?
You will encounter when assigned merge commit to head. (note that it's not always. please try some commit.)
I suggest to change [L100](https://github.com/storz/prdiff/pull/2/files#diff-9ba84b6b4500a897458d3976afde5bb2R100) into like below
```
if mergedAt.Equal(t1) || mergedAt.After(t1) && mergedAt.Before(t2) || mergedAt.Equal(t2) { // [t1, t2]
```
or something simplified one...

Reply:
@storz I fixed it following the comment. PTAL 🙇